opus/application/static_media/js/detail.js

```diff
@@ -208,4 +217,19 @@ var o_detail = {
         }
     },
 
+    // Adjust the positions of detail preview image nav buttons "<" & ">" to make them
+    // stay right next to the image at both sides.
+    adjustDetailImgNavBtns: function() {
+            let detailPreviewImgWdith = $(".op-detail-img img").width();
+            let offset = 40;
```

Comment:
why is offset 40 (40... pixels...?)

Reply:
By default when css left & right of "<" & ">" are 0, those two buttons stay within the image container with their arrow tips touching the left & right borders of the container (see the attached pic). 
The width of "<" & ">" is bit more than 25px.
To move buttons out of the image, we need to set css left & right to at least -(26px + some values) so that buttons won't be too close the image.
Therefore we pick 40 here.
![Screen Shot 2022-07-18 at 3 10 33 PM](https://user-images.githubusercontent.com/17075939/179625902-ef48ec79-b08a-49a0-85c6-2716973469b4.png)

Reply:
I think a brief comment would be great regarding your reasoning.

Reply:
It seems like Dave already added these comments a couple of days ago